DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/Index.cshtml

```diff
@@ -31,27 +31,7 @@
     <h2 class="nhsuk-heading-m">Usage stats</h2>
 
     <div class="js-only-block">
-      <ul class="legend nhsuk-body-s nhsuk-u-margin-bottom-0">
-        <li class="ct-series-a nhsuk-u-margin-right-2 nhsuk-u-margin-bottom-0">
-          <svg height="10" width="40" class="nhsuk-u-margin-right-1">
-            <line x1="0" y1="0" x2="40" y2="0" class="ct-line legend-line" />
-          </svg>
-          Completions
-        </li>
-        <li class="ct-series-b nhsuk-u-margin-right-2 nhsuk-u-margin-bottom-0">
-          <svg height="10" width="40" class="nhsuk-u-margin-right-1">
-            <line x1="0" y1="0" x2="40" y2="0" class="ct-line legend-line" />
-          </svg>
-          Evaluations
-        </li>
-        <li class="ct-series-c nhsuk-u-margin-right-2 nhsuk-u-margin-bottom-0">
-          <svg height="10" width="40" class="nhsuk-u-margin-right-1">
-            <line x1="0" y1="0" x2="40" y2="0" class="ct-line legend-line" />
-          </svg>
-          Registrations
-        </li>
-      </ul>
-      <div class="ct-chart ct-perfect-fourth"></div>
+      <partial name="_ActivityGraph.cshtml"/>

```

Comment:
Isn't this a "legend" rather than graph?

Reply:
It also contains the graph itself, it felt appropriate to keep it all in one unit since the graph is a small component html-wise and the legend should ideally be part of it.

Reply:
Ah - yes, I missed the graph div in the partial! In that case, it makes sense as is 👍